.eslintrc.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 6,
     ecmaFeatures: {
-      jsx: true
+      jsx: true,
+      experimentalObjectRestSpread: true
```

Comment:
It allows the use of the `spread` operator in creating new objects without it getting pinged by the linter. I used it a fair amount in the reducers.